src/test/java/net/minecraftforge/debug/LootDebug.java

```diff
@@ -0,0 +1,77 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.entity.item.EntityMinecartChest;
+import net.minecraft.entity.monster.EntityCreeper;
+import net.minecraft.entity.monster.EntitySkeleton;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.LootEntry;
+import net.minecraft.world.storage.loot.LootEntryItem;
+import net.minecraft.world.storage.loot.LootPool;
+import net.minecraft.world.storage.loot.RandomValueRange;
+import net.minecraft.world.storage.loot.conditions.LootCondition;
+import net.minecraft.world.storage.loot.functions.LootFunction;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.LootGenerateEvent;
+import net.minecraftforge.event.LootTableLoadEvent;
+import net.minecraftforge.event.entity.living.LivingDropsEvent;
+import net.minecraftforge.event.entity.minecart.MinecartInteractEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+
+@Mod(modid = "forgelootdebug")
+public class LootDebug {
+
+    @Mod.EventHandler
+    public void preInit(FMLPreInitializationEvent evt) {
+        MinecraftForge.EVENT_BUS.register(this);
+    }
+
+    @SubscribeEvent
+    public void fishingHook(LootTableLoadEvent evt)
+    {
+        System.out.println(evt.getPath().toString());
+        if ("minecraft:gameplay/fishing".equals(evt.getPath().toString()))
+        {
+            evt.addPool(new LootPool(new LootEntry[]{
+                    new LootEntryItem(Items.cooked_porkchop, 200, 1, new LootFunction[0], new LootCondition[0])
+            }, new LootCondition[0], new RandomValueRange(3), new RandomValueRange(0)));
```

Comment:
Oh god this is some ugly code, maybe we should look into builders?
Or better yet, NOT have them build these items in memory instead load then from disk...